Userland/Libraries/LibGfx/Matrix.h

```diff
@@ -60,6 +60,9 @@ class Matrix {
     constexpr auto elements() const { return m_elements; }
     constexpr auto elements() { return m_elements; }
 
+    constexpr auto const& operator()(size_t row, size_t col) const { return m_elements[row][col]; }
+    constexpr auto& operator()(size_t row, size_t col) { return m_elements[row][col]; }
```

Comment:
If this is too weird I can change it to a more traditional `at()` method. What I really want is `operator[]`, but that's C++23 only, and I guess we're still on C++20

Reply:
We could probably bump to C++23, might need a minimum compiler version update though?
cc @ADKaster ?

Reply:
Still fixme territory, we're still waiting on oss-fuzz to update